openmetadata-service/src/main/java/org/openmetadata/service/resources/search/SearchResource.java

```diff
@@ -150,7 +150,13 @@ public Response search(
               description =
                   "Get only selected fields of the document body for each hit. Empty value will return all fields")
           @QueryParam("include_source_fields")
-          List<String> includeSourceFields)
+          List<String> includeSourceFields,
```

Comment:
can we keep this method in glossaryResource instead of searchResource?

Reply:
/api/v1/glossary/hierachy?fqn=glossaryName

Reply:
currently we are supporting hierarchy only on search we need to have it in searchResource , later on if we want to extend it to other entities like team, it will be much easier with SearchResource

Reply:
@sonika-shah I rather have deidcated apis, we may have others that we use it here, but centralizing it in search is not clear in the api path